src/components/document/document.tsx

```diff
@@ -213,7 +208,7 @@ export class DocumentComponent extends BaseComponent<IProps, IState> {
   private renderProblemTitleBar(type: string, hideButtons?: boolean) {
     const {problem, appMode, clipboard, user: { isTeacher }} = this.stores;
     const problemTitle = problem.title;
-    const {document, workspace} = this.props;
+    const {document, workspace } = this.props;
```

Comment:
```suggestion
    const { document, workspace } = this.props;
```
If you're going to add the one space you should probably add the other. 😉